synapse_rock/rockcraft.yaml

```diff
@@ -129,4 +126,18 @@ parts:
         plugin: dump
         source: attributemaps
         organize:
-            login_ubuntu.py: /usr/local/attributemaps/login_ubuntu.py
\ No newline at end of file
+            login_ubuntu.py: /usr/local/attributemaps/login_ubuntu.py
+    synapse-start:
+        plugin: dump
+        after:
+          - synapse
+        source: .
+        # start.py is extracted from
+        # https://github.com/matrix-org/synapse/blob/develop/docker/start.py
+        # Our version does not use chown command that, depending of the /data
+        # size, could take a long time to run.
+        organize:
+            start_custom.py: start.py
```

Comment:
Would it be better to "patch" the original one, so we can see when it is changed upstream?

Reply:
Do you mean patching in the rockcraft itself? Using sed commands or something similar?

Reply:
But then I would need to git clone synapse repository right? Right now the start.py is obtained from what pip installs.